src/pudl/transform/ferc1.py

```diff
@@ -6141,3 +6172,135 @@ def plants_steam_ferc1(
         transformed_fuel=fuel_ferc1,
     )
     return convert_cols_dtypes(df, data_source="ferc1")
+
+
+def get_other_dimensions() -> list[str]:
+    """Get a list of the other dimension columns across all of the transformers."""
+    # grab all of the dimensions columns that we are currently verifying as a part of
+    # reconcile_table_calculations
+    other_dimensions = []
+    for transformer in FERC1_TFR_CLASSES.values():
+        other_dimensions.append(
+            transformer().params.reconcile_table_calculations.subtotal_column
+        )
+    # remove nulls and dedupe
+    other_dimensions = [sub for sub in other_dimensions if sub]
+    other_dimensions = list(set(other_dimensions))
+    return other_dimensions
```

Comment:
this is now kiiiind of a dupe of `other_dimensions` over int the outputs.. i could add a `table_names=None` which defaults to all of the tables to reuse this in both places.